tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -15,197 +15,431 @@
-          "course_list": [
-            "POL 230",
-            "POL 349",
-            "POL 351",
-            "POL 352",
-            "POL 355",
-            "POL 356",
-            "POL 360",
-            "POL 362",
-            "POL 364",
-            "POL 366",
-            "POL 367",
-            "POL 374",
-            "POL 375",
-            "POL 378",
-            "POL 384",
-            "POL 386",
-            "POL 430",
-            "POL 431",
-            "POL 432",
-            "POL 433",
-            "POL 434",
-            "POL 479"
+          "name": "Primary Field",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "Concentrators must take at least three courses in their primary field. The primary field may be changed at any time before the end of the first semester of the junior year.",
+          "req_list": [
+            {
+              "name": "Political Theory",
+              "max_counted": 1,
+              "min_needed": 3,
+              "explanation": "The PT sub-field focuses on the nature of justice, democracy, power, and other key ideas, and encourages students to develop frameworks for thinking evaluatively about pressing issues of politics and public policy of the day.",
+              "course_list": [
+                "POL 210",
+                "POL 301",
+                "POL 303",
+                "POL 305",
+                "POL 307",
+                "POL 308",
+                "POL 311",
+                "POL 315",
+                "POL 316",
+                "POL 317",
+                "POL 319",
+                "POL 406",
+                "POL 418",
+                "POL 473",
+                "POL 474",
+                "POL 475",
+                "POL 492"
+              ]
+            },
+            {
+              "name": "American Politics",
+              "max_counted": 1,
+              "min_needed": 3,
+              "explanation": "The AP sub-field focuses on the U.S. political system and includes the study of the development of the American system of governance, American political institutions, the attitudes and behaviors of U.S. residents, and the relationship between institutions and people.",
+              "course_list": [
+                "POL 220",
+                "POL 315",
+                "POL 316",
+                "POL 318",
+                "POL 319",
+                "POL 320",
+                "POL 324",
+                "POL 325",
+                "POL 326",
+                "POL 327",
+                "POL 329",
+                "POL 330",
+                "POL 334",
+                "POL 335",
+                "POL 336",
+                "POL 339",
+                "POL 341",
+                "POL 342",
+                "POL 344",
+                "POL 349",
+                "POL 420",
+                "POL 422",
+                "POL 423",
+                "POL 428",

```

Missing, based on [this list](https://politics.princeton.edu/undergraduate/complete-course-listing?field-of-study%5B18%5D=18). It's a new course this Fall.
```suggestion
                "POL 426",
                "POL 428",
```